src/main/java/sirius/biz/process/ProcessController.java

```diff
@@ -238,6 +239,12 @@ public void toggleDebugging(WebContext webContext, String processId) {
     @Permission(PERMISSION_MANAGE_PROCESSES)
     public void updatePersistencePeriod(WebContext webContext, String processId) {
         Process process = findAccessibleProcess(processId);
+        if (process != null && !process.canChangePersistencePeriod()) {
```

Comment:
SonarLint ist quite stupid. It requires me to make a null check since `process` could be null (ain't since the method will thrown an exception, but the compiler does not know that).

The thing is: process is used below, why it never complaint before? Just because it's used in an `if`? 🤔